### ISSUE TITLE
SUGGESTION: Energy Bar Ticker (video)

### ISSUE THREAD

**Issue by Jackodawacko** [Resike/Z-Perl#12]
Hi there,

I promise this will be the last thing I add since I know I've posted a bit on GitHub haha. I'm wondering if you might be able to add the energy tick bar for rogues like how it was in original X-Perl? I've listed the Youtube video that I took years ago which shows it. The way they had it was super clean and seeing if there's a chance you would be able to replicate it like how the look is in the original? 

https://www.youtube.com/watch?v=cnYDsnXMNi8
 
Thank you for listening and sorry again for the tickets lately!

**Comment by Resike** [comment]
Yeah I know that was a thing back then, I just need to dug up that code back from X-Perl.

**Comment by Jackodawacko** [comment]
If you want, once I get back home in a few weeks, I can dig up the old
add-on I had on my external hard drive if that helps?

**Comment by Macumbafeh** [comment]
:0 i asked it before and didn't get an answer for the same thing !
Don't forget the 5 second rule bar ! :)

**Comment by Macumbafeh** [comment]
:0 i asked it before and didn't get an answer for the same thing !
Don't forget the 5 second rule bar ! :)
